llvm-spirv/lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -840,7 +840,9 @@ SPIRVFunction *LLVMToSPIRVBase::transFunctionDecl(Function *F) {
     if (Attrs.hasParamAttr(ArgNo, Attribute::ReadOnly))
       BA->addAttr(FunctionParameterAttributeNoWrite);
     if (Attrs.hasParamAttr(ArgNo, Attribute::ReadNone))
-      BA->addAttr(FunctionParameterAttributeNoReadWrite);
+      // TODO: intel/llvm customization
```

Comment:
I think that not adding an attribute at all (removing the if block) might be a better idea here. Does that cause some issues?